tests/run/cpp_exception_declaration_compatibility.srctree

```diff
@@ -35,4 +41,5 @@ cdef class TestClass:
 
 cdef class TestClass:
     cpdef test_func(self) except +handle_exception:
+        cppfunc()  # GH 3064 - cppfunc caused invalid code to be generated with +handle_exception
```

Comment:
This probably isn't _quite_ what the test is about, but it seemed the closest related place that it could be tested.